lib/3scale_toolbox/commands/copy_command/copy_service.rb

```diff
@@ -40,7 +40,8 @@ def run
             Tasks::CopyApplicationPlansTask.new(context),
             Tasks::CopyLimitsTask.new(context),
             Tasks::DestroyMappingRulesTask.new(context),
-            Tasks::CopyMappingRulesTask.new(context)
+            Tasks::CopyMappingRulesTask.new(context),
+            Tasks::CopyPricingRulesTask.new(context)
```

Comment:
The same comment as 
https://github.com/3scale/3scale_toolbox/pull/101#discussion_r263277216

Reply:
:+1: